### DISCUSSION TITLE
Series of UI Icons Needed :)

### DISCUSSION THREAD

**Discussion by SinaTaavon** [twilio-labs/paste#2024] [Request: Icon] [answered]
Hey all,

We are developing a live conference platform using Paste (and a series of other Twilio services) and to keep things standard we need a couple of UI icons, here is the exact list.

- [Microphone On](https://app.streamlinehq.com/icons/streamline-mini-line/computer-devices/voice-mail/computer-voice-mail) 
- [Microphone Off](https://app.streamlinehq.com/icons/streamline-mini-line/computer-devices/voice-mail/computer-voice-mail-off)
- [Video On](https://app.streamlinehq.com/icons/streamline-regular/work-office-companies/video-meetings/meeting-camera)
- [Video Off](https://app.streamlinehq.com/icons/streamline-regular/video-movies-tv/controls/controls-camera-off)
- [Share Screen](https://app.streamlinehq.com/icons/streamline-regular/interface-essential/share/share-1)
- [Stop Sharing Screen](https://app.streamlinehq.com/icons/streamline-regular/interface-essential/multimedia-controls/button-stop)
- [Raise Hand](https://app.streamlinehq.com/icons/streamline-regular/interface-essential/select/cursor-hand-2)
- [Lower Hand](https://app.streamlinehq.com/icons/streamline-regular/wayfinding/allowances/allowances-no-talking)

I could not find anything better than a single share button for the "Screen Share" button. I will leave the decision making to you guys if you have a more fitting icon for that operation I would love to use that one either.

Here is a screenshot of the application and where these icons would go:

![image](https://user-images.githubusercontent.com/32191839/141133171-a6d6499a-b7e5-443e-a890-078e21ec41e1.png)

At the moment we are using the MUI Icons but we would like to use Paste for standardisation purposes.

Please let me know if you need any more information :)

**Comment by nkrantz** [comment]
Hi @SinaTaavon, we've created [a ticket for these icons](https://issues.corp.twilio.com/browse/DSYS-3130) and we'll update this thread once they're added to the system. Thanks!

**Comment by loreina** [comment, marked as answer]
Hi @SinaTaavon, thanks so much for your patience. We added the icons in #2059 and they should be added to Paste in our next release! 🎉
